src/HealthChecks.AzureServiceBus/ClientCache.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Concurrent;
+
+namespace HealthChecks.AzureServiceBus;
+
+internal static class ClientCache
```

Comment:
I suggest to move it into source-shared class (from CosmosDb) + remove another copy from HealthChecks.AzureStorage. Example of source-shared class is `<Compile Include="../CallerArgumentExpressionAttribute.cs" Link="CallerArgumentExpressionAttribute.cs" />` in Directory.Build.props. You may include this file under condition by `MSBuildProjectName` (if you like) to not clutter other projects.

Reply:
Moved it and scoped it in Directory.Build.props to the three projects using ClientCache, AzureServiceBus, AzureStorage and CosmosDb. However, I have not changed to any async usages in AzureStorage or CosmosDb. Nor have I added ClientCache in the AzureServiceBus* classes. Don't know if I should or not, but I'm fine with that being a new PR 😄